_tpl/_html-foot.tpl

```diff
@@ -18,54 +18,7 @@
 <script src="{{ url static_file='_js/fancybox/jquery.mousewheel-3.0.4.pack.js' }}" type="text/javascript"></script>
 <script src="{{ url static_file='_js/fancybox/jquery.fancybox-1.3.4.pack.js' }}" type="text/javascript"></script>
 <link rel="stylesheet" type="text/css" href="{{ url static_file='_js/fancybox/jquery.fancybox-1.3.4.css' }}" media="screen" />
-<a id="fancybox-trigger" href="#werbung-mitglied" style="display:none"></a>
-<script type="text/javascript">
-        function triggerWerbungPopup() {
-            $("#fancybox-trigger").fancybox({
```

Comment:
Best would to remove eveything. (Yeah i know i said something else before, but i was lazy.) Just write down the commits where you remove it and then once we need it again, we can revert the commits.